### DISCUSSION TITLE
Fresh install of KlipperScreen via WAUIH fails with 'could not activate the environment, try deleting /home/pi/.KlipperScreen-env and retry"

### DISCUSSION THREAD

**Discussion by paynterf** [KlipperScreen/KlipperScreen#1492] [Q&A]
Following this YouTube video, using WAUIH I installed Klipper, Moonraker, Mainsail and Fluidd successfully on a Rpi 3B+.  Because I later plan to add the ELECROW 5 inch Monitor, I also attempted to install KlipperScreen.  Unfortunately, the KlipperScreen install failed with the error message noted in the title.  

I couldn't find KlipperScreen.log (and couldn't figure out the steps to produce it), I have included a text file showing the console log.  Any ideas would be appreciated.
[241024_KIAUH_Install_Failure.txt](https://github.com/user-attachments/files/17528135/241024_KIAUH_Install_Failure.txt)

**Comment by alfrix** [comment]
it's repeated multiple times in the log:
```
Error writing to file - write (28: No space left on device)
```
that would explains the issue installing

**Comment by paynterf** [comment]
Thanks!  That was the problem - I was using a 4GB SD, thinking "well, that
should be PLENTY of space!" - not 😭

32GB SD card arriving later today

Frank


On Sat, Oct 26, 2024 at 10:23 AM Alfredo Monclus ***@***.***>
wrote:

> it's repeated multiple times in the log:
>
> Error writing to file - write (28: No space left on device)
>
> that would explains the issue installing
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/KlipperScreen/KlipperScreen/discussions/1492#discussioncomment-11060673>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA6T325TJUFPLFYTOAHUAGLZ5OQX5AVCNFSM6AAAAABQUE34U6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBWGA3DOMY>
> .
> You are receiving this because you authored the thread.Message ID:
> <KlipperScreen/KlipperScreen/repo-discussions/1492/comments/11060673@
> github.com>
>


-- 
G.Frank Paynter, PhD
OSU ESL Research Scientist (ret)
EM Workbench LLC
614 638-6749 (cell)
